go.mod

```diff
@@ -8,15 +8,15 @@ require (
 	github.com/cyphar/filepath-securejoin v0.2.3
 	github.com/distribution/distribution/v3 v3.0.0-20221119093643-85d4039064cc
 	github.com/fluxcd/go-git-providers v0.11.0
-	github.com/fluxcd/go-git/v5 v5.0.0-20221104190732-329fd6659b10
+	github.com/fluxcd/go-git/v5 v5.0.0-20221206140629-ec778c2c37df
```

Comment:
We need to keep this in-sync with https://github.com/fluxcd/flux2/blob/main/tests/azure/go.mod#L7